src/styles/utils/index.ts

```diff
@@ -253,6 +253,10 @@ function getAvatarBorderRadius(size: AvatarSizeName, type?: string): ViewStyle {
         return {borderRadius: avatarBorderSizes[size]};
     }
 
+    if (type === CONST.ICON_TYPE_SQUARE) {
```

Comment:
I'm afraid it might break avatars in the app, why do you need this if statement?

Reply:
I need it to set borderRadius to 0 but right now it's the only case it's needed; I can only choose between having `borderRadius:100` which is too much or `borderRadius:50` bundled with non-transparent background. IMHO this is the least invasive way.